jaxrs-spec/src/main/asciidoc/boilerplate.adoc

```diff
@@ -0,0 +1,45 @@
+﻿//
+//    Copyright (c) 2019 Markus KARG
+//
+//    This program and the accompanying materials are made available under the
+//    terms of the Eclipse Public License v. 2.0, which is available at
+//    http://www.eclipse.org/legal/epl-2.0.
+//
+//    This Source Code may also be made available under the following Secondary
+//    Licenses when the conditions for such availability set forth in the
+//    Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+//    version 2 with the GNU Classpath Exception, which is available at
+//    https://www.gnu.org/software/classpath/license.html.
+//
+//    SPDX-License-Identifier: EFSL-1.0
+//
+
+== Boilerplate
+
+WARNING: This document contains just boilerplate information. It explicitly is *not* a complete specification. In particular, it misses all normative text. To learn about compliance with this specification, refer to "JAX-RS: Java™ API for RESTful Web Services" (Version 2.1, Final Release July 13, 2017).
+
+
+=== Scope
+
+*Jakarta RESTful Web Services* provides a specification document, TCK and foundational API to develop web services following the Representational State Transfer (REST) architectural pattern.
+
+
+=== Acronyms
+
+Acronyms used in this document MAY refer to Oracle / JCP technology.
+
+The acronym JAX-RS is reused with permission of Oracle and either refers to "Jakarta RESTful Web Services" or "Java™ API for RESTful Web Services", dependending on the context.
```

Comment:
I'm not sure that's going to be acceptable to Oracle legal, who wants a very clear
distinction between the use of the acronym to refer to the JCP spec and the use
of the acronym to refer to the Eclipse spec.

As currently written, I'm not sure this spec needs to use the acronym at all, and
it certainly doesn't need to use it to refer to the JCP spec.

Reply:
I own written permission by Oracle legal to use the acronym as it is *not* an Oracle trademark.

Currently the branch `EE4J_8` still contains the string "JAX-RS" a lot of times, so we still need the clarification in the boilerplate spec. Once the acronym is not used anymore (if ever, as there is no legal need to remove it), we certainly will drop this line.

Reply:
The "disclaimer" about the acronym reference is supposed to be in the "document"
that contains the reference.  Adding the disclaimer to the spec document when there
are no references in the spec document is pointless.

Since the acronym is used in the source code, adding the disclaimer to the source
repository, e.g., in the NOTICE file or the package javadocs, would be appropriate.

Still, as written, your disclaimer doesn't follow the guidelines that were agreed to.

Reply:
Understood.

Can you please post a link to that guidelines?

Reply:
Sorry, I thought this had already been widely circulated.  The guidelines are [here](https://jakarta.ee/legal/acronym_guidelines/).

Reply:
PR #779 drops the use of the acronym in sources and related files